EIPS/eip-###.md

```diff
@@ -0,0 +1,224 @@
+---
+eip: <to be assigned>
+title: Multi-Fractional Non-Fungible Token Standard
+description: 
+author: David Kim(@powerstream3604)
+discussions-to: https://github.com/ethereum/EIPs/issues/4674
```

Comment:
`discussions-to` should point to a forum thread at https://ethereum-magicians.org/, not a GitHub issue.

Out of curiosity, where did you see that a GitHub issue should be created?  We recently changed the policy on this and if there is some place we forgot to update it would be great to find and fix it!

Reply:
I think I was not updated about that.
I created a topic in ethereum magicians and added that
topic link for "discussions to" link.